### PR TITLE
[ISSUE-11] Add banner

### PR DESCRIPTION
**Pull request by pooja-amura**
- make changes in banner 
- change div element with react-bootstrap

Screenshot from 2020-04-24 12-48-02

### CODE REVIEW

File: src/components/Banner.js [hunk 1]
```diff
@@ -1,55 +1,18 @@
...
-        </a>
-        <a className="carousel-control-next" href="#myCarousel" role="button" data-slide="next">
-          <span className="carousel-control-next-icon" aria-hidden="true"></span>
-          <span className="sr-only">Next</span>
-        </a>
-      </div>
+      <Row className="mt-5 mb-5">
+        <Col>
+          <Alert variant="info">
+            <marquee>
```
Comment (huzefa-amura) [inline src/components/Banner.js:10]: @pooja-amura : We this work as a quick flash for some seconds or will rotate of right to left ? We don't need the later one as the quotes should just flash in a span period of 5 seconds.

**Comment by huzefa-amura** [comment]
@pooja-amura : We also need the footer to be fixed at the bottom of the page regardless of the data being present in the table, it should remain at the bottom of the page only. 
Currently it shifts to upper side if data not present which shouldn't happen, can you please fix this ?

**Comment by pooja-amura** [comment]
> @pooja-amura : We also need the footer to be fixed at the bottom of the page regardless of the data being present in the table, it should remain at the bottom of the page only.
> Currently it shifts to upper side if data not present which shouldn't happen, can you please fix this ?

Done

File: src/components/Banner.js (hunk 1 above)
Comment (pooja-amura) [reply src/components/Banner.js:10]: lets have a call for this

Comment (huzefa-amura) [reply src/components/Banner.js:10]: @pooja-amura : Sure.

**Pull request merged**